controller/iexInterface.js

```diff
@@ -64,8 +62,30 @@ iexRequest['income'] = (symbol) => { return axios.get(iexRequest.parseUrl(`stock
 iexRequest['financials'] = (symbol) => { return axios.get(iexRequest.parseUrl(`stock/${symbol}/financials/last`)); }
 
 // Chart quote requests
-iexRequest.test.intraday = () => { return axios.get(iexRequest.parseTestUrl("stock/aapl/intraday-prices")); }
-iexRequest.test.historicalPrices = () => { return axios.get(iexRequest.parseTestUrl("stock/aapl/chart/1m")); }
+iexRequest.test.historicalPrices1d = async () => {
+    const historicalPriceResponse = await axios.get(iexRequest.parseTestUrl(`stock/aapl/chart/1d`));
+    return historicalPriceResponse.data;
+}
+
+iexRequest.test.historicalPrices1m = async () => {
+    const historicalPriceResponse = await axios.get(iexRequest.parseTestUrl(`stock/aapl/chart/1m`));
+    return historicalPriceResponse.data;
+}
+
+iexRequest.test.historicalPrices3m = async () => {
+    const historicalPriceResponse = await axios.get(iexRequest.parseTestUrl(`stock/aapl/chart/3m`));
+    return historicalPriceResponse.data;
+}
+
+iexRequest.test.historicalPrices1y = async () => {
+    const historicalPriceResponse = await axios.get(iexRequest.parseTestUrl(`stock/aapl/chart/1y`));
+    return historicalPriceResponse.data;
+}
+
+iexRequest.test.historicalPrices5y = async () => {
+    const historicalPriceResponse = await axios.get(iexRequest.parseTestUrl(`stock/aapl/chart/5y`));
+    return historicalPriceResponse.data;
```

Comment:
These 5 functions are exactly the same, could you make use of period parameter and make this a single function please